opentelemetry-api/src/opentelemetry/propagators/composite.py

```diff
@@ -80,3 +82,8 @@ def fields(self) -> typing.Set[str]:
                 composite_fields.add(field)
 
         return composite_fields
+
+
+@deprecated(version="1.2.0", reason="You should use CompositePropagator")  # type: ignore
+class CompositeHTTPPropagator(CompositePropagator):
+    """Alias for CompositePropagator"""
```

Comment:
```suggestion
    """CompositeHTTPPropagator provides a mechanism for combining multiple
    propagators into a single one.
```

Reply:
Not sure I follow, wouldn't we want the documentation to point users to the non-deprecated class instead?

Reply:
The documentation of each class should just document each class. `"""Alias for CompositePropagator"""` is not documenting the `CompositeHTTPPropagator` class. Warning about a class being deprecated is a separate issue and it is the responsibility of the `deprecated` decorator.